### DISCUSSION TITLE
Upgrading to 0.14.0 breaks

### DISCUSSION THREAD

**Discussion by eddyinet** [wireui/wireui#126] [Q&A] [answered]
I just updated to v0.14.0 to take advantage of some fixes and also updated to Alpine 3 (3.5.1) as well as it's a requirement. I'm on LiveWire 2.7 and after updating all my WireUI components won't work. I see the scripts are loading correctly but running for example a dialog or notification comes up empty. No errors in the console or logs. I tried manually running from the console for example:

`window.$wireui.notify({
    title: 'Profile saved!',
    description: 'Your profile was successfull saved',
    icon: 'success'
})`

Response is simply : undefined

Just want to double check I'm not on any conflicting versions or if anyone else has seen something similar?

Thanks!

**Comment by ph7jack** [comment]
Can you list all components that are not working?

You already have a notifications component imported?
https://livewire-wireui.com/docs/notifications#first-steps

**Reply by eddyinet** [reply]
Sure. So I'm only using x-modal.card, dialog and notifications and all 3 don't work. I've confirmed they're all imported and I see them loading in the source.

**Reply by eddyinet** [reply]
So just for kicks I added a <x-inputs.phone label="Phone" /> to a blank LiveWire view just to see if it renders and I get this error:

`Undefined array key "label"`

`Illuminate\Foundation\Bootstrap\HandleExceptions::handleError
vendor/ph7jack/wireui/src/WireUiComponentResolver.php:11
 
namespace WireUi;
 
class WireUiComponentResolver
{
    public function resolve(string $originalComponentName): string
    {
        $components = config('wireui.components');
 
        return $components[$originalComponentName]['alias'];
    }
}
 
`

**Reply by vlados** [reply]
What is in your components array in config/wireui.php?
Did you updated the config after the update?

**Reply by eddyinet** [reply]
Yup.... I did:

'components' => [
        'icon' => [
            'class' => Components\Icon::class,
            'alias' => 'icon',
        ],
        'icon.spinner' => [
            'class' => Components\Icons\Spinner::class,
            'alias' => 'icon.spinner',
        ],
        'input' => [
            'class' => Components\Input::class,
            'alias' => 'wireinput',
        ],
        'textarea' => [
            'class' => Components\Textarea::class,
            'alias' => 'textarea',
        ],
        // 'label' => [
        //     'class' => Components\Label::class,
        //     'alias' => 'label',
        // ],
        'error' => [
            'class' => Components\Error::class,
            'alias' => 'error',
        ],
        'errors' => [
            'class' => Components\Errors::class,
            'alias' => 'errors',
        ],
        'inputs.maskable' => [
            'class' => Components\Inputs\MaskableInput::class,
            'alias' => 'inputs.maskable',
        ],
        'inputs.phone' => [
            'class' => Components\Inputs\PhoneInput::class,
            'alias' => 'inputs.phone',
        ],
        'inputs.currency' => [
            'class' => Components\Inputs\CurrencyInput::class,
            'alias' => 'inputs.currency',
        ],
        'button' => [
            'class' => Components\Button::class,
            'alias' => 'wirebutton',
        ],
        'dropdown' => [
            'class' => Components\Dropdown::class,
            'alias' => 'dropdown',
        ],
        'dropdown.item' => [
            'class' => Components\Dropdown\DropdownItem::class,
            'alias' => 'dropdown.item',
        ],
        'dropdown.header' => [
            'class' => Components\Dropdown\DropdownHeader::class,
            'alias' => 'dropdown.header',
        ],
        'notifications' => [
            'class' => Components\Notifications::class,
            'alias' => 'notifications',
        ],
        'datetime-picker' => [
            'class' => Components\DatetimePicker::class,
            'alias' => 'datetime-picker',
        ],
        'time-picker' => [
            'class' => Components\TimePicker::class,
            'alias' => 'time-picker',
        ],
        'card' => [
            'class' => Components\Card::class,
            'alias' => 'card',
        ],
        'native-select' => [
            'class' => Components\NativeSelect::class,
            'alias' => 'native-select',
        ],
        'select' => [
            'class' => Components\Select::class,
            'alias' => 'select',
        ],
        'select.option' => [
            'class' => Components\Select\Option::class,
            'alias' => 'select.option',
        ],
        'select.user-option' => [
            'class' => Components\Select\UserOption::class,
            'alias' => 'select.user-option',
        ],
        'toggle' => [
            'class' => Components\Toggle::class,
            'alias' => 'toggle',
        ],
        'checkbox' => [
            'class' => Components\Checkbox::class,
            'alias' => 'checkbox',
        ],
        'radio' => [
            'class' => Components\Radio::class,
            'alias' => 'radio',
        ],
        'modal' => [
            'class' => Components\Modal::class,
            'alias' => 'modal',
        ],
        'modal.card' => [
            'class' => Components\ModalCard::class,
            'alias' => 'modal.card',
        ],
        'dialog' => [
            'class' => Components\Dialog::class,
            'alias' => 'dialog',
        ],
    ],

**Comment by eddyinet** [comment, marked as answer]
Solved! You lead me down the right path. For anyone else encountering this:

If you're using webpack with Laravel and upgrading from Alpine 2 to Alpine 3, in your app.js you can't just do:

`require('alpinejs');`

but as per the doc:

```
import Alpine from 'alpinejs'
window.Alpine = Alpine
Alpine.start()
```

Thanks @PH7-Jack and @vlados !

Great library... thank you!
